dio/lib/src/options.dart

```diff
@@ -336,52 +343,55 @@ class _RequestConfig {
     this.method,
     this.receiveTimeout,
     this.sendTimeout,
-    Map<String, dynamic> extra,
-    Map<String, dynamic> headers,
-    String contentType,
+    Map<String, dynamic>? extra,
+    Map<String, dynamic>? headers,
+    String? contentType,
```

Comment:
```suggestion
    this.contentType,
```

Reply:
contentType is not a class field bur set/get

Reply:
You're obviously right!